deploy/addon/postgresql/pgpool.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Pgpool-II for PostgresSQL add-ons
+position: 3
+shortdesc: How to configure and use Pgpool-II for PostgresSQL add-ons
+tags:
+- addons
+keywords:
+- balancing
+- connection
+- limit
+- load
+- pgpool
+- pool
+- postgres
+- proxy
+- sql
+---
+
+Pgpool-II is a tool available on application instances to help you manage your connection pool and load balancing of your PostgreSQL add-ons.
+
+{{< alert "warning" "Warning:" >}}
+Pgpool-II is not available on Docker instances. If you want to use Pgpool-II, you can add it and configure it in your Dockerfile.
+{{< /alert >}}
+
+## What is Pgpool-II?
+
+{{< readfile "/content/partials/pgpool.md" >}}
+
+## Why use Pgpool-II?
+
+**Connection pooling**: Pgpool-II maintains established connections to the PostgreSQL servers, and reuses them whenever a new connection with the same properties (i.e. user name, database, protocol version, and other connection parameters if any) comes in. It reduces the connection overhead, and improves the system's overall throughput.
+
+**Load balancing**: If a database is replicated (because running in either replication mode or native replication mode), performing a SELECT query on any server will return the same result. Pgpool-II takes advantage of the replication feature in order to reduce the load on each PostgreSQL server. It does that by distributing SELECT queries among available servers, improving the system's overall throughput. In an ideal scenario, read performance could improve proportionally to the number of PostgreSQL servers. Load balancing works best in a scenario where a lot of users execute many read-only queries at the same time.
```

Comment:
```suggestion
**Load balancing**: If a database is replicated (because running in either replication mode or native replication mode), performing a `SELECT` query on any server will return the same result. Pgpool-II takes advantage of the replication feature in order to reduce the load on each PostgreSQL server. It does that by distributing `SELECT` queries among available servers, improving the system's overall throughput. In an ideal scenario, read performance could improve proportionally to the number of PostgreSQL servers. Load balancing works best in a scenario where a lot of users execute many read-only queries at the same time.
```